Jenkinsfile

```diff
@@ -125,7 +125,7 @@ def checkoutCentreonBuild(buildBranch) {
 /*
 ** Pipeline code.
 */
-stage('Source') {
+stage('Deliver sources') {
```

Comment:
Jenkinsfile modifications are out of the PR description scope.
Could you rebase your work ?

Reply:
I pushed a commit in order to avoid rpm builded from CI branches to go on unstable repo.